src/script/deploy.js

```diff
@@ -0,0 +1,283 @@
+  }
+}
+
+///////////////////////////////////////////////////////////////////
+///                            UTILS                            ///
+///////////////////////////////////////////////////////////////////
+async function loadConfiguration(useConfig) {
+  if (!useConfig) {
+    return {};
+  }
+  let answer = await ask(`Do you want to load configuration from prior runs? [Y/n]: `, "bool");
+  const spinner = ora("Configuration Loading").start();
+  if (answer === undefined) {
+    answer = true;
+  }
+  if (answer) {
+    if (!fs.existsSync(CONFIG_FILENAME)) {
+      spinner.warn("Configuration load requested but no configuration available: continuing");
+      return {};
+    }
+    try {
+      const fileContents = JSON.parse(fs.readFileSync(CONFIG_FILENAME).toString());
+      if (fileContents) {
+        spinner.succeed("Configuration loaded");
+        return fileContents;
+      } else {
+        spinner.warn("Unable to parse configuration: deleting and continuing");
+        fs.rmSync(CONFIG_FILENAME);
+        return {};
+      }
+    } catch {
+      spinner.warn("Unable to parse configuration: deleting and continuing");
+      fs.rmSync(CONFIG_FILENAME);
+      return {};
+    }
+  } else {
+    spinner.succeed("Configuration not loaded");
+    return {};
+  }
+}
+
+async function saveConfiguration(config) {
+  const oldData = (() => {
+    try {
+      return JSON.parse(fs.readFileSync(CONFIG_FILENAME).toString());
+    } catch {
+      return {};
+    }
+  })();
+
+  const data = JSON.stringify({ ...oldData, ...config });
+  fs.writeFileSync(CONFIG_FILENAME, data);
+}
+
+///////////////////////////////////////////////////////////////////
+///                         DEPLOYMENTS                         ///
+///////////////////////////////////////////////////////////////////
+
+async function deployLineaWorldID(config) {
+  const spinner = ora("Deploying LineaID on Linea...").start();
+
+  try {
+    const data = execSync(
+      `forge script src/script/DeployLineaWorldID.s.sol:DeployOpWorldID --fork-url ${config.lineaRpcUrl} \
+      --etherscan-api-key ${config.lineaEtherscanApiKey} --broadcast --verify -vvvv`,
+    );
+    console.log(data.toString());
+  } catch (err) {
+    console.error(err);
+  }
+
+  spinner.succeed("DeployLineaWorldID.s.sol ran successfully!");
+}
+
+///////////////////////////////////////////////////////////////////
+///                      MAINNET DEPLOYMENT                     ///
+///////////////////////////////////////////////////////////////////
+
+async function deployLineaStateBridgeMainnet(config) {
+  const spinner = ora("Deploying Linea State Bridge...").start();
+
+  try {
+    const data =
+      execSync(`forge script src/script/DeployLineaStateBridgeMainnet.s.sol:DeployLineaStateBridgeMainnet --fork-url ${config.ethereumRpcUrl} \
+      --etherscan-api-key ${config.ethereumEtherscanApiKey} --broadcast --verify -vvvv`);
```

Comment:
I guess we need to provide the address of `LineaWorldID`, which was deployed earlier